### ISSUE TITLE
[Idea] Filter nodes

### ISSUE THREAD

**Issue by LorenzHenk** [tchayen/markdown-links#71]
When having a lot of markdown files with lots of links, things can get messy and the overview the graph adds declines.

More often than not, you're interested in the relations of the current file.
It would be cool to be able to filter node and only show nodes that are connected to the current file by N edges (where N is configurable in the graph view).

That way you can remove most clutter in the graph by setting N to e.g. 1.

**Comment by deconDarts** [comment]
I think this falls into a place where there is substantial overlap with issue requests.

maybe a project roadmap?

PR #50 appears to start this for #38 Adjacency Graph & for IS #7 

IS #63 custom search subset is IS #47 link to Sections

IS #17 Color Coding

I came to "Filter Nodes" to check on searching unlinked .md files in the folder/tree; and, perused.
Interesting to consider a file/tree [hierarchy] view w/ linking representations.

hope this isn't too brief of a consolidation of thought; a bit of a tangent from current endeavor.

**Comment by tchayen** [comment]
@deconDarts absolutely, many requests are now overlapping and or could use some broader vision.

#50 and #38 are more of implementation details.

#63, #47 and #48 indeed would be nice to handle them at once.

Not sure what and how to do it though, I will have a look around other issues and PRs that happened during my absence and will try to come up with a plan.
